metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/src/main/config/zookeeper/enrichments/bro.json

```diff
@@ -0,0 +1,17 @@
+{
+    "enrichment" : {
```

Comment:
Will it?  Where is the logic that installed the enrichments before?  Can't we just let that existing logic install the enrichments?